config/jobs/kubernetes/sig-testing/kubernetes-kind.yaml

```diff
@@ -138,6 +140,14 @@ presubmits:
           requests:
             cpu: 4
             memory: 9Gi
+      # use experimental new nodepool with C4 VM, cgroup v2
```

Comment:
this is the prod job 🤦 

testing it anyhow, this job has been busted regardless

Reply:
https://prow.k8s.io/view/gs/kubernetes-ci-logs/pr-logs/pull/126563/pull-kubernetes-e2e-kind-ipv6/1926052312141271040

is working, so i'm inclined to roll forward

Reply:
roll-forward in https://github.com/kubernetes/test-infra/pull/34841